server/sandbox_run_linux.go

```diff
@@ -771,13 +771,18 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 		if hostNetwork {
 			g.RemoveMount("/sys")
 			g.RemoveMount("/sys/cgroup")
-			sysMnt := spec.Mount{
+			g.AddMount(spec.Mount{
 				Destination: "/sys",
 				Type:        "bind",
 				Source:      "/sys",
 				Options:     []string{"nosuid", "noexec", "nodev", "ro", "rbind"},
-			}
-			g.AddMount(sysMnt)
+			})
+			g.AddMount(spec.Mount{
```

Comment:
`/sys/fs/cgroup` will be mounted as part of the `/sys` bind mount (it has `rbind` - recursive bind).  Is there any reason to create a new mount here?

Reply:
I can't remember why this was here... I think I was making an attempt